### ISSUE TITLE
Dropdown items - Links - Wrong color

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#434]
**Describe the bug**
when links are rendered in the dropdown component, they are incorrectly displayed as blue.  Not just blue, but our focus blue which fails accessibility contrast ratio for text.

![image](https://user-images.githubusercontent.com/24397282/150995467-6f9948da-ad50-44f5-8ae4-242dcb11c9a1.png)

**To Reproduce**
The bug can be viewed here: https://react-magma.cengage.com/version/2.5.6/api/dropdown/#dropdownmenunavitem

**Expected behavior**
Menu item links should display the same as non-link menu items - Dark gray

![image](https://user-images.githubusercontent.com/24397282/150995665-2c2b809c-a482-44dc-bf03-efc4372193da.png)
